app/src/main/resources/db/migration/V1__Tabeller.sql

```diff
@@ -40,7 +40,8 @@ CREATE TABLE behandling
     beslutter               VARCHAR                  NULL,
     iverksattTidspunkt      TIMESTAMP WITH TIME ZONE NULL,
     sist_endret             TIMESTAMP WITH TIME ZONE NOT NULL,
-    opprettet               TIMESTAMP WITH TIME ZONE NOT NULL
+    opprettet               TIMESTAMP WITH TIME ZONE NOT NULL,
+    vilkårssett             JSONB                    NOT NULL
```

Comment:
Jeg sa ja til json i db, men jeg må innrømme at du har lagt det "høyere opp" enn jeg regnet med, jeg hadde ikke trodd at hele vilkårssettet skulle bli en stor json-blob. Jeg trodde det ville være litt mer "blandings"..

Reply:
Hvis har behov for spørringer på et mer granulært nivå, kan vi migrere deler av dette.